sdk/include/psdk/ntgdi.h

```diff
@@ -1986,7 +1986,7 @@ NtGdiGetCharWidthW(
     _In_ HDC hdc,
     _In_ UINT wcFirst,
     _In_ UINT cwc,
-    _In_reads_opt_(cwc) PWCHAR pwc,
+    _In_reads_opt_(cwc) const WCHAR *pwc,
```

Comment:
```suggestion
    _In_reads_opt_(cwc) PCWCH pwc,
```
or PCWSTR if it is null-terminated (but not sure: it seems to be counted)